garage/np/algos/base.py

```diff
@@ -5,3 +5,17 @@ class Algorithm:
 class RLAlgorithm(Algorithm):
     def train(self):
         raise NotImplementedError
+
```

Comment:
please make `RLAlgorithm` an ABC and use `pass` and `@abstractmethod` to define these.

Reply:
i don't see a fix here

Reply:
generally we wait for the reviewer to resolve any comments, not the author :)

Reply:
I think this is an outdated version. This is fixed.